src/generated/resources/openapi.yaml

```diff
@@ -741,6 +741,30 @@ components:
       format: date
       type: string
       example: 2022-03-10
+    DeserializerTech:
```

Comment:
I feel this enum models [(data) formats](https://docs.confluent.io/platform/current/schema-registry/fundamentals/serdes-develop/index.html#supported-formats) instead of something specific to deserialization. WDYT?